classes/student.rb

```diff
@@ -0,0 +1,13 @@
+require './classes/person'
+
+# Define a sub-base class
+class Student < Person
+  def initialize(classroom, age)
+    super(age)
```

Comment:
- [ ] Kindly make sure to call the `super` method correctly with the right parameters according to the parent's constructor. The `initialize` method here should accept all of the parameters to pass down to the `super` constructor.

```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)
```